Cargo.lock

```diff
@@ -4,9 +4,9 @@ version = 3
 
 [[package]]
 name = "addr2line"
-version = "0.17.0"
```

Comment:
[minor-1]
probably no need to change `Cargo.lock`, I reverted it in [mvlookup_integration](https://github.com/scroll-tech/halo2/commits/mvlookup_integration) branch